transitions/index.js

```diff
@@ -29,70 +29,35 @@ var availableTransitions = [
   'update_sent_forms'
 ];
 
-/*
- * Supported configuration:
```

Comment:
Ok I added configuration examples to the readme, should be all set now. https://github.com/medic/medic-sentinel/blob/2207-disable-transitions-by-default/README.md